docker-compose.yml

```diff
@@ -10,12 +10,21 @@ services:
       POSTGRES_USER: ${DB_USERNAME:-safira}
       POSTGRES_PASSWORD: ${DB_PASSWORD:-secret}
       POSTGRES_PORT: ${DB_PORT:-5432}
-      POSTGRES_HOST: ${DB_HOST:-localhost}
     ports:
       - ${DB_PORT:-5432}:5432
+    expose:
+      - 5432
     volumes:
       - pgdata:/var/lib/postgresql/data
     restart: always
+  web:
+    build: 
+      context: .
+      dockerfile: Dockerfile
+    env_file: .env
+    ports:
+      - 4000:4000
```

Comment:
Wouldn't we benefit from having a volume that keeps our directory and docker equal? Instead of rerun every time we change something?

Reply:
[Docker Sync](https://docker-sync.readthedocs.io/en/latest/) could be a solution for that

Reply:
Yes, for testing purposes mounting a volume to not rebuild the entire image would be nice. I will edit the compose file